SIG-Readme-Template.md

```diff
@@ -0,0 +1,17 @@
+# <Name> Special Interest Group
+<Goals of this SIG>
+
+## Team
+* <Member Name> (@Github id), <Company Name>
+* <Member Name> (@Github id), <Company Name>
+* <Member Name> (@Github id), <Company Name>
+
+## Meeting Info
+* Meeting Name
+  * Dial in info
+  * [Meeting Minutes](URL).
```

Comment:
I'd suggest to put it on top

Reply:
this is kind of what people are checking out before deciding whether to join the meeting. Not other way around